### PR TITLE
SB-857 login controller

### PR DESCRIPTION
**Pull request by fuss86**

### CODE REVIEW

File: strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/configuration/Configuration.java [hunk 1]
```diff
@@ -43,6 +43,9 @@
     @XmlElement(name = "proxy-configuration")
     private ProxyConfiguration proxyConfiguration;
 
+    @XmlElement(name = "session-configuration")
```
Comment (carlspring) [inline strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/configuration/Configuration.java]: Thanks for adding this! :)

File: strongbox-web-core/src/main/java/org/carlspring/strongbox/security/authentication/suppliers/AuthenticationSupplier.java [hunk 2]
```diff
@@ -19,4 +20,8 @@
     @CheckForNull
     Authentication supply(@Nonnull HttpServletRequest request);
 
+    default boolean supports(@Nonnull HttpServletRequest request) {
```
Comment (carlspring) [inline strongbox-web-core/src/main/java/org/carlspring/strongbox/security/authentication/suppliers/AuthenticationSupplier.java]: Cosmetic: Could you please bump the curly brace on a new line?

**Comment by carlspring** [comment]
Looks good, let me know, when it's ready for merging!
;)

**Comment by fuss86** [comment]
@carlspring it's ready :)

**Pull request merged**